code/modules/modular_computers/file_system/programs/security/penalmechs.dm

```diff
@@ -0,0 +1,113 @@
+/datum/computer_file/program/penal_mechs
+	filename = "penalmechs"
+	filedesc = "Penal Mech Monitoring"
```

Comment:
This should probably be "Remote Penal Monitoring", since more things than mechs will likely appear on this program once the time comes.

Reply:
Changed to that.